src/firebaseConfig.js

```diff
@@ -0,0 +1,22 @@
+import { initializeApp } from 'firebase/app';
+import { getAnalytics } from 'firebase/analytics';
+import { getFirestore } from 'firebase/firestore';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyChtlkduXmo0XbrofVuHAsMDVQzM3hKumk',
```

Comment:
better keep API key abstracted like in `.env` file

Reply:
Yeah, i have already done this change in my other branch